Assets/Scripts/Actors/ActorComponents/PlayerActorPhysics.cs

```diff
@@ -34,7 +32,11 @@ public override void SetupStateMethodMap()
 
 	void Jumping()
 	{
-		JumpCheck();
+		// TODO should we be doing JumpCheck() while jumping?
```

Comment:
GetButtons during FixedUpdate sometimes miss button presses. Should change this to set a flag every frame if input is down.